utils/swift_build_support/swift_build_support/products/llvm.py

```diff
@@ -274,10 +274,12 @@ def build(self, host_target):
         llvm_cmake_options.define('COMPILER_RT_INTERCEPT_LIBDISPATCH', 'ON')
 
         llvm_enable_projects = ['clang']
+        llvm_enable_runtimes = []
 
         if self.args.build_compiler_rt and \
                 not self.is_cross_compile_target(host_target):
-            llvm_enable_projects.append('compiler-rt')
+            llvm_enable_runtimes.append('compiler-rt')
+            build_targets.append("runtimes")
```

Comment:
This is causing macOS tests to fail, since we add this even when we are skipping builds

```
/Users/ec2-user/jenkins/workspace/swift-PR-macos/branch-main/swift/validation-test/BuildSystem/skip_build_xcode.test:14:15: error: CHECK-NEXT: expected string not found in input
# CHECK-NEXT: env {{.+}}/cmake --build {{.+}}/Xcode-ReleaseAssert/llvm-macosx-{{.+}} --config Release --target ZERO_CHECK -- {{.*}}-target llvm-tblgen -target clang-resource-headers -target intrinsics_gen -target clang-tablegen-targets{{$}}
              ^
...
        16: + env /usr/local/Cellar/cmake/3.24.2/bin/cmake --build BUILD_DIR/validation-test-macosx-x86_64/BuildSystem/Output/skip_build_xcode.test.tmp/Xcode-ReleaseAssert/llvm-macosx-x86_64 --config Release --target ZERO_CHECK -- -parallelizeTargets -jobs 12 -target llvm-tblgen -target clang-resource-headers -target intrinsics_gen -target clang-tablegen-targets -target runtimes 

```